go.mod

```
@@ -0,0 +1,11 @@
+module go-rejson
```

```suggestion
module github.com/nitishm/go-rejson
```
This way the indirect self-dependency of the project will be resolved.